apps/labs/posts/jupyterlab-winter-theme.md

```diff
@@ -0,0 +1,347 @@
+to start to change the overall color scheme before doing more detailed
+customisation.
+
+We'll try to update the current theme from orange and blue to more blue-ish tones, 
+which tend to remind us of the holiday
+season.  Afterward, in the `diff`, see how we changed some of the colors:
+
+```diff
+diff --git a/style/variables.css b/style/variables.css
+index f8c738f..9f39f14 100644
+--- a/style/variables.css
++++ b/style/variables.css
+@@ -87,9 +87,9 @@ all of MD as it is not optimized for dense, information rich UIs.
+    */
+
+   --jp-border-width: 1px;
+-  --jp-border-color0: var(--md-grey-400);
+-  --jp-border-color1: var(--md-grey-400);
+-  --jp-border-color2: var(--md-grey-300);
++  --jp-border-color0: #168EA8;
++  --jp-border-color1: #168EA8;
++  --jp-border-color2: #93D3E1;
+   --jp-border-color3: var(--md-grey-200);
+   --jp-border-radius: 2px;
+
+@@ -118,10 +118,10 @@ all of MD as it is not optimized for dense, information rich UIs.
+```
+
+We'd like the "running notebook" dot in the filesystem browser to be a
+snowman instead of a blue dot. Using the browser inspector, we can look at the CSS
+doing this and override it in my theme:
+
+```diff
+@@ -368,0 +369,26 @@                                                                  
++
++.jp-DirListing-item.jp-mod-running .jp-DirListing-itemIcon:before {
++    content: '\2603'; /* snowman */
++    font-size: 10px;
++    position: absolute;
++    left: -8px;
++}
+```
+
+We can also add backgrounds to many of the panels.  By adding a file `snowflakepatterns.svg` to our style directory,
+we can now refer to it from our file and add the following CSS to include snowflakes in the background of
+our directory listing.
+
+```css
+/* DirListing */
+
+body[data-jp-theme-name="JupyterLab Winter"] .jp-DirListing-content {
+  background-image: url('./snowflakepattern.svg');
+ }
+```
+
+# Awesome!
+
+[Here](https://github.com/Quansight-Labs/jupyterlab-theme-winter) is the final result, jupyterLab-theme-winter theme provided by Quansight.  Feel
+free to modify it, and please suggest some themes you might like and share
+in the comments. For example we'd love to see a "summer 2020 theme" for our Southern Hemisphere friends.
+We will dive into how to distribute your themes and make them high-quality in a later blog post.
+
+[![Screenshot of JupyterLab Winter Theme](/posts/jupyterlab-winter-theme/jupyterlab-theme-winter.png)](https://github.com/Quansight-Labs/jupyterlab-theme-winter)
+
+And as a bonus, a [Christmas theme](https://github.com/Quansight-Labs/jupyterlab-theme-christmas) with more green-ish color and some lights shining
+at the bottom of your notebooks!
+
+[![Screenshot of JupyterLab Christmas Theme](/posts/jupyterlab-winter-theme/jupyterlab-theme-christmas.png)](https://github.com/Quansight-Labs/jupyterlab-theme-christmas)
+
+---
+
+*This is part of a series of Jupyter tutorials. Find more [tutorials here](/categories/jupytertutorials).*
```

Comment:
```suggestion
```

I'm taking this out because the only link at that category on the old site is to this blog post.